ci/scripts/quick-bazel.sh

```diff
@@ -10,28 +10,67 @@
 
 set -e -x
 
+GCP_BAZEL_CACHE_KEY=$1
+
 mkdir -p bazel-results
 
+# Install bazel
+# TODO: this can be deleted once #12210 is merged
+./bazelisk.sh
+
+# Call bazel in all subsequent commands using bazelisk
```

Comment:
Is there a particular reason to have a separate env variable.

Reply:
Just for readability